templates/pages/login.html.twig

```diff
@@ -34,6 +34,10 @@
 {% extends 'layout/page_card_notlogged.html.twig' %}
 
 {% block content_block %}
+   {% if redirect %}
+   {{ call('Toolbox::manageRedirect', [redirect]) }}
```

Comment:
`Toolbox::manageRedirect` will redirect the user, unless I am wrong.

Reply:
I mean, it will output a `location` header or using a `window.location=xxx` script then exit. I am not sure it should be call in the middle of a twig template.

Reply:
@orthagh should I put this back in ```index.php```

Reply:
> I mean, it will output a `location` header or using a `window.location=xxx` script then exit. I am not sure it should be call in the middle of a twig template.

Actually work locally and from internal instance (BTW)

Reply:
Agree with @cedric-anne, sorry for the double change of mind

Reply:
(no pb) done !